src/app/[locale]/layout.tsx

```diff
@@ -51,6 +52,23 @@ export default function RootLayout({children, params}: Props) {
   return (
     <html className="h-full" lang={locale}>
       <body className="flex flex-col min-h-screen">
+        <div className="sr-only">
+          <ul>
+            <li>
+              <a href="#facets">{tLanguageScreenreaderMenu('jumpFilters')}</a>
+            </li>
+            <li>
+              <a href="#search-results">
+                {tLanguageScreenreaderMenu('jumpResults')}
+              </a>
+            </li>
+            <li>
+              <a href="#pagenavigation">
```

Comment:
Can you choose one method for word separation? Now I see "#search-results" and "#pagenavigation".

Reply:
ok